main.go

```diff
@@ -236,7 +240,24 @@ func main() {
 	// Start the web server.
 	gin.SetMode(gin.ReleaseMode)
 	r := gin.New()
-	external.InitPrometheus(r) // init prometheus if enabled
+
+	// choose the statsCollector we will use.
+	if cfg.Prometheus.Enabled {
```

Comment:
Could we move that into a private helper function? To make main flow more clean? Maybe only personal preference :sweat_smile:

Reply:
👍 I moved it into stats_collector